Community.cpp

```diff
@@ -24,6 +24,23 @@
 
 #include "Community.h"
 
+#ifdef _OPENMP
+#ifdef __has_include
+#if __has_include(<version>)
+#include <version>
+#endif
+#endif
+#include <atomic>
+#if __cpp_lib_barrier >= 201907L && __cpp_lib_optional >= 201606L
+#include <barrier>
+#include <optional>
+#else
+#include <condition_variable>
+#include <mutex>
+#endif
+#include <omp.h>
+#endif // _OPENMP
+
```

Comment:
```suggestion
#ifdef _OPENMP

#ifdef __has_include
#if __has_include(<version>)
#include <version>
#endif
#endif

#include <atomic>

#if __cpp_lib_barrier >= 201907L && __cpp_lib_optional >= 201606L
#include <barrier>
#include <optional>
#else
#include <condition_variable>
#include <mutex>
#endif

#include <omp.h>

#endif // _OPENMP
```

Adding a few line breaks for readability